pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_prerequisites.adoc

```diff
@@ -12,14 +12,16 @@
 
 = Prerequisites
 
+You must have the following environment to use {ProductName}. 
+
 .Operating Systems
 
 [NOTE]
 ====
-Operating system support affects the performance of the uploader more than other Pantheon components.
+Operating system support affects the performance of the uploader more than other {ProductName} components.
```

Comment:
I think that this was written before we were told that there is no uploader tool anymore. @benradey @emarcusRH ?

Reply:
Good idea asking the questions from here down.

Once we answer them, let's raise a new issue to make these changes so that we can focus on editorial aspects in this PR.